osu.Game/Screens/OnlinePlay/Lounge/Components/RoomsContainer.cs

```diff
@@ -265,5 +239,11 @@ protected override void Dispose(bool isDisposing)
             if (roomManager != null)
                 roomManager.RoomsUpdated -= updateSorting;
         }
+
+        public void HideAnyPopovers()
+        {
+            // must be called on a child of the PopoverContainer due to parent traversal not considering self.
+            roomFlow.HidePopover();
+        }
```

Comment:
I think this can be a direct call now that the popover container is at base-game level?